provision/acc_provision/acc_provision.py

```diff
@@ -1123,6 +1191,67 @@ def generate_rancher_yaml(config, operator_output, operator_tar, operator_cr_out
             template.stream(config=config).dump(operator_output)
 
 
+def is_calico_flavor(flavor):
+    flav = SafeDict(FLAVORS[flavor])
+    ovl = flav["calico_cni"]
+    if ovl is True:
+        return True
+
+    return False
```

Comment:
this function implementation is not very pythonic, would just a:

    return SafeDict(FLAVORS[flavor]).get("calico_cni")

not have worked?

Reply:
sure, that works. Thanks